src/dists.ad.js

```diff
@@ -505,15 +778,15 @@ function expGammaSample(shape, scale) {
 function expGammaScore(shape, scale, val) {
   'use ad';
   var x = val;
-  return (shape - 1) * x - Math.exp(x) / scale - logGamma(shape) - shape * Math.log(scale);
+  return (shape - 1) * x - Math.exp(x) / scale - Math.logGamma(shape) - shape * Math.log(scale);
```

Comment:
Where is `Math.logGamma` defined? I see `logGamma` in `special.js` - do we extend the `Math` object somewhere?